### DISCUSSION TITLE
Showing Invalid Configuration when added costume domain

### DISCUSSION THREAD

**Discussion by theprasant** [vercel/community#240] [Help] [answered]
Hello, I tried to add a costume domain. But it's showing `invalid Configuration` 

![error](https://user-images.githubusercontent.com/67827472/132244035-f4bc56c9-e32d-4df1-8243-2cbd9ad273c2.jpg)

**Comment by leerob** [comment, marked as answer]
Looks like it might be an issue with Cloudflare and your configuration to it - did you check that learn more link?

**Reply by theprasant** [reply]
> Looks like it might be an issue with Cloudflare and your configuration to it - did you check that learn more link?

Thanks for responding.
I asked `js.org` team and they added it to CNAME and the problem was solved. Although it's still showing `Invalid Configuration`, I have no problem with it.
Thank you.

**Comment by DeadstarIII** [comment]
same here
